packages/CalendarCarousel/index.tsx

```diff
@@ -289,6 +352,17 @@ function CalendarCarousel<T>({
           }}
           keyExtractor={(item, id): string => id.toString()}
         />
+        <FlatList
+          style= {styles.eventContainer}
+          data= {dayEvents}
+          numColumns = {1}
+          renderItem={({ item }): ReactElement => {
+            if (eventSwitch) {
+              return (
+                <Text>{item}</Text>);
+            }
+          }}
+        />
```

Comment:
No `FlatList` for footers. Use `ListFooterComponent` instead with one `FlatList`.

Reply:
Should I do the same for the header part?
(The View element above the flatlist)
<img width="581" alt="Screen Shot 2020-08-03 at 10 17 35 AM" src="https://user-images.githubusercontent.com/59305253/89137280-93bf5780-d572-11ea-884a-7ea99c90fcce.png">

Reply:
Well no. If you put them in `ListHeaderComponent`, they will also be scrollable and will have double vertical scroll behavior when the users wrap that with `<FlatList` outside the component. This is an anti-pattern.

To think of it again, I think `eventList` should be rendered outside the `FlatList`. Sorry about this. 

Also, please name it `{ renderEvents() }`

Reply:
Should I wrap a view around this ? (<View> </View>)

<img width="851" alt="Screen Shot 2020-08-03 at 12 37 30 PM" src="https://user-images.githubusercontent.com/59305253/89143601-24536300-d586-11ea-8c78-d4edb412ee80.png">

The renderEvents currently looks like this 
<img width="648" alt="Screen Shot 2020-08-03 at 12 38 31 PM" src="https://user-images.githubusercontent.com/59305253/89143644-40ef9b00-d586-11ea-874e-8913048d6ea2.png">

Reply:
This looks great except that you used `FlatList` again. Do not use `FlatList` here and rather use the map and draw elements. This is not a good UX to provide multiple scrolls in one screen.

Reply:
Yup fixed it !